src/status_im2/common/scroll_page/view.cljs

```diff
@@ -41,65 +28,98 @@
+  (let [scroll-height (reagent/atom negative-scroll-position-0)
+        input-range   (if platform/ios? [-67 10] [0 150])]
     (fn [sticky-header children]
-      [:<>
-       [:<>
-        [rn/image
-         {:source   cover
-          :position :absolute
-          :style    (style/image-slider (get-header-size @scroll-height))}]
-        [rn/blur-view (style/blur-slider (get-header-size @scroll-height))]]
-       [rn/view {:style {:z-index 6 :margin-top (if platform/ios? 44 0)}}
-        [quo/page-nav
-         {:horizontal-description? true
-          :one-icon-align-left?    true
-          :align-mid?              false
-          :page-nav-color          :transparent
-          :page-nav-background-uri ""
-          :mid-section             {:type            :text-with-description
-                                    :main-text       (when (>= @scroll-height scroll-position-1) name)
-                                    :description-img (when (>= @scroll-height scroll-position-1) icon)}
-          :right-section-buttons   (:right-section-buttons page-nav)
-          :left-section            {:icon                  :i/close
-                                    :icon-background-color (icon-color)
-                                    :on-press              #(rf/dispatch [:navigate-back])}}]
-        (when sticky-header [sticky-header @scroll-height])]
-       [rn/scroll-view
-        {:style (style/scroll-view-container (diff-with-max-min @scroll-height 16 0))
-         :shows-vertical-scroll-indicator false
-         :scroll-event-throttle 4
-         :on-scroll #(swap! scroll-height (fn [] (int (oops/oget % "nativeEvent.contentOffset.y"))))}
-        [rn/view {:style {:height 151}}
-         [rn/image
-          {:source cover
-           :style  {:overflow :visible
-                    :flex     1}}]]
-        (when children
-          [rn/view
-           {:flex             1
-            :border-radius    (diff-with-max-min @scroll-height 16 0)
-            :background-color (colors/theme-colors
-                               colors/white
-                               colors/neutral-90)}
-           [children @scroll-height icon-top-fn icon-size-fn]])]])))
+      [:f>
+       (fn []
+         (let [y         (animated/value @scroll-height)
+               animation (animated/interpolate y
+                                               {:inputRange  input-range
+                                                :outputRange [1.2 0.5]
+                                                :extrapolate (:clamp animated/extrapolate)})]
+           [:<>
+            [scroll-page-header scroll-height name page-nav cover sticky-header]
+            [rn/scroll-view
+             {:style                           (style/scroll-view-container
+                                                (diff-with-max-min @scroll-height 16 0))
+              :shows-vertical-scroll-indicator false
+              :scroll-event-throttle           4
```

Comment:
Hmm, what about phones supporting 120fps? Should we use 8ms?

Reply:
Good question, I think testing different throttle settings in such phones would be ideal, and measure the app's performance.

But this throttle setting is just about triggering the scroll event handler, our app still needs to do a lot of processing to re-render after the scroll event happens, so I doubt we're getting 60fps anyway nowadays. That's also why we have to be careful when lowering the throttle setting, because it can actually make things worse.

Reply:
In summary, it's the complete rendering path that should be measured. Lowering the throttle setting too much can put too much pressure on the rendering pipeline (re-frame, React, RN) and reduce the overall snappiness of the app.

Reply:
Hmm.. I think we need to investigate more into this. Also I think we can have a condition to lower the throttle depending on the device's refresh rate

Reply:
Yep, the impact of this setting needs to be better understood, but for sure our RN app is not capable of keeping a 16ms average to render the entire stack top to bottom, or at least I highly doubt that.